examples/node/index.ts

```diff
@@ -21,46 +25,49 @@ async function demo() {
     return
   }
   console.log('>> Setup Wallet')
-  const provider = new providers.JsonRpcProvider(
-    'https://polygon-rpc.com/',
-    137
-  )
+
+  const myRpcProviders = [
+    {
+      chainId: 43114,
+      url: 'https://open-platform.nodereal.io/959d0cc48937455d937a1a52ef1d503f/avalanche-c/ext/bc/C/rpc',
+    },
+    {
+      chainId: 43114,
+      url: 'https://avax-mainnet.gateway.pokt.network/v1/lb/6303a5e60295e8003b5bce00',
+    },
+  ]
+
+  const provider = new RPCProvider(myRpcProviders)
+
   const wallet = Wallet.fromMnemonic(mnemonic).connect(provider)
 
   // get Route
-  console.log('>> Request route')
+  console.log('>> Configuring route')
   const routeRequest = {
-    fromChainId: ChainId.POL, // Polygon
-    fromAmount: '1000000', // 1 USDT
-    fromTokenAddress: findDefaultToken(CoinKey.USDT, ChainId.POL).address,
-    toChainId: ChainId.DAI, // xDai
-    toTokenAddress: findDefaultToken(CoinKey.USDT, ChainId.DAI).address,
+    fromChainId: ChainId.AVA, // Avalanche
+    fromAmount: '100000', // 1 USDT
+    fromTokenAddress: findDefaultToken(CoinKey.USDC, ChainId.AVA).address,
+    toChainId: ChainId.AVA, // Avalanche
+    toTokenAddress: findDefaultToken(CoinKey.USDT, ChainId.AVA).address,
     options: {
       slippage: 0.03, // = 3%
       allowSwitchChain: false, // execute all transaction on starting chain
-      exchanges: {
-        allow: [], // only find direct transfers
-      },
+      // exchanges: {
+      //   allow: [], // only find direct transfers
+      // },
     },
   }
 
   // STEP 1: Initialize the API
 
   // ☝️ This configuration is totally optional! ------------------------------------
   const optionalConfigs: ConfigUpdate = {
-    apiUrl: 'https://li.quest', // DEFAULT production endpoint
-    rpcs: {
-      // You can provide custom RPCs
-      137: ['https://polygon-rpc.com/'],
-    },
-    multicallAddresses: {
-      // You can provide custom addresses for multicall
-      137: '0x02817C1e3543c2d908a590F5dB6bc97f933dB4BD',
-    },
+    integrator: 'lifi-sdk', // DEFAULT 'lifi-sdk'
```

Comment:
can we find a better integrator string? "lifi-sdk-sample" for example?

Reply:
just something that hints that the request comes from the example

Reply:
also the js file has no integrator?

Reply:
Added, it was present already but the push failed.